tests/polling/infrequent/workflow_test.py

```diff
@@ -10,7 +10,7 @@
 
 
 async def test_infrequent_polling_workflow(client: Client, env: WorkflowEnvironment):
-    if not env.supports_time_skipping:
+    if env.supports_time_skipping:
         pytest.skip("Too slow to test with time-skipping disabled")
```

Comment:
The boolean condition and the message don't seem to jointly make sense any longer?

Reply:
Ah, yes, it looks like the original message should have been "enabled" (in addition to the conditional), will fix that

Reply:
Hrmm, maybe I misunderstood the situation here and we are wanting to only run this test with time skipping and it's still too slow even with time skipping. Will look.